css/selectors/marker.json

```
@@ -49,7 +49,7 @@
               "version_added": false
             },
             "safari": {
-              "version_added": false
+              "version_added": "11.1"
             },
             "safari_ios": {
               "version_added": false
```

Since we're here, I did a little digging and found the changeset for this: https://trac.webkit.org/changeset/220207/webkit That maps to Safari for iOS 11.3:

```suggestion
              "version_added": "11.3"
```